skimage/transform/_warps.py

```diff
@@ -648,22 +650,22 @@ def _clip_warp_output(input_image, output_image, order, mode, cval, clip):
 
     Other parameters
     ----------------
-    order : int, optional
+    order : int
```

Comment:
We no more need the `order` argument here now :wink:

Reply:
True, I'm correcting this :wink: